great_expectations/data_context/migrator/cloud_migrator.py

```diff
@@ -188,18 +180,60 @@ def _emit_warnings(
             self._warn_about_bundle_contains_datasources()
 
     def _warn_about_test_migrate(self) -> None:
-        pass
+        logger.warning(
+            "This is a test run! Please pass `test_migrate=False` to begin the "
+            "actual migration (e.g. `CloudMigrator.migrate(context=context, test_migrate=False)`).\n"
+        )
 
     def _warn_about_usage_stats_disabled(self) -> None:
-        pass
+        logger.warning(
+            "We noticed that you had disabled usage statistics tracking. "
+            "Please note that by migrating your context to GX Cloud your new Cloud Data Context "
+            "will emit usage statistics. These statistics help us understand how we can improve "
+            "the product and we hope you don't mind!\n"
+        )
 
     def _warn_about_bundle_contains_datasources(self) -> None:
-        pass
+        logger.warning(
+            "Since your existing context includes one or more datasources, "
+            "please note that if your credentials are included in the datasource config, "
+            "they will be sent to the GX Cloud backend. We recommend storing your credentials "
+            "locally in config_variables.yml or in environment variables referenced "
+            "from your configuration rather than directly in your configuration. Please see "
+            "our documentation for more details.\n"
+        )
 
     def _print_configuration_bundle_summary(
         self, configuration_bundle: ConfigurationBundle
     ) -> None:
-        pass
+        # NOTE(cdkini): Datasources should be verbose, not just summary!
+        to_print = (
+            ("Datasource", configuration_bundle.datasources),
```

Comment:
This needs to be updated to give a verbose output - will update in a subsequent PR